libs/blocks/merch/merch.js

```diff
@@ -376,8 +413,9 @@ export async function getCommerceContext(el, params) {
 }
 
 /**
- * Checkout parameter can be set Merch link, code config (scripts.js) or be a default from tacocat.
- * To get the default, 'undefinded' should be passed, empty string will trigger an error!
+ * Checkout parameter can be set on the merch link,
+ * code config (scripts.js) or be a default from tacocat.
+ * To get the default, 'undefined' should be passed, empty string will trigger an error!
```

Comment:
is this a lint change? (fine with me, just clarifying)

Reply:
yes

Reply:
also typo: undefinded :D

Reply:
hehe